src/main/java/de/chojo/repbot/dao/snapshots/RepProfile.java

```diff
@@ -90,10 +90,12 @@ private EmbedBuilder getBaseBuilder(Configuration configuration, LocalizationCon
 
         var currProgress = String.valueOf(reputation() - currentRoleRep);
         var nextLevel = nextRoleRep.equals(currentRoleRep) ? "Ꝏ" : String.valueOf(nextRoleRep - currentRoleRep);
+
+        var title = localizer.localize("element.profile.title", Replacement.create("NAME", repUser.member().getEffectiveName()));
+
         var build = new LocalizedEmbedBuilder(localizer)
-                .setAuthor("%s%s".formatted(rank() != 0 ? "#" + rank() + " " : "", "element.profile.title"),
```

Comment:
Quoting the key with $ would be already enough :D

`"%s$%s$"`

Reply:
Well, thats the easier solution